### DISCUSSION TITLE
Amaze File Converter

### DISCUSSION THREAD

**Discussion by shuvashish76** [TeamAmaze/AmazeFileManager#3274] [Ideas]
Discussion for https://github.com/TeamAmaze/AmazeFileUtilities-Issue-Tracker/issues/24

**Comment by EmmanuelMess** [comment]
This is not an AmazeFileManager issue.

**Comment by VishalNehra** [comment]
I've enabled discussion on amaze utils issue tracker repo
https://github.com/TeamAmaze/AmazeFileUtilities-Issue-Tracker/discussions
